src/sorcha/modules/PPFootprintFilter.py

```diff
@@ -70,7 +107,7 @@ def __init__(self, points, ID=0, units="radians"):
         self.centerx = np.sum(self.x) / len(self.x)
         self.centery = np.sum(self.y) / len(self.y)
 
-    def ison(self, point, ϵ=10.0 ** (-11), plot=False):
+    def ison(self, point, ϵ=10.0 ** (-11), edge_thresh=None, plot=False):
```

Comment:
I know that you didn't add it, but I wish that the original developer has used `epsilon`...